frontend/package.json

```diff
@@ -68,7 +68,7 @@
     "@scalprum/core": "^0.1.2",
     "@testing-library/dom": "^8.13.0",
     "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.3.0",
+    "@testing-library/react": "12.1.5",
```

Comment:
@karelhala mind chiming in here for historical purposes why we needed to downgrade? IIRC when I looked version 13 may not have been compatible with all the tools we use.